packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -176,13 +176,26 @@ export default class SupportNewIssuesFormController {
       name: `answer-finding-${isSuccess ? 'yes' : 'no'}`,
       type: 'action',
     });
+
+    const service = {
+      ...this.currentServiceName,
+      serviceName: this.serviceType.name === 'kube'
+        ? this.currentServiceName.url.match(/\/([a-f0-9]*)\//)[1]
+        : this.currentServiceName.serviceName,
+    };
+
+    const impactedResource = this.serviceType.name === 'kube'
+      ? this.currentServiceName.url.match(/\/((?:[a-f0-9]*-)+[a-f0-9]*)\//)[1]
```

Comment:
fyi you also have the stateParams attributes that contains [projectId, kubeId], so it's easier to fetch the parameters from this array

Reply:
Indeed 😄